sample-config.js

```diff
@@ -216,7 +216,8 @@ config.trader = {
   key: '',
   secret: '',
   username: '', // your username, only required for specific exchanges.
-  passphrase: '' // GDAX, requires a passphrase.
+  passphrase: '', // GDAX, requires a passphrase.
+  currencyAmountAvailableToTrade = false; // set to a number to limit the amount of fiat currency available to Gekko to trade. Set to false to disable.
```

Comment:
does this also need to go into https://github.com/askmike/gekko/blob/stable/config/plugins/trader.toml to set the defaults? not sure how these toml files work.